onnxruntime/test/perftest/test_configuration.h

```diff
@@ -74,6 +75,10 @@ struct PerformanceTestConfig {
   ModelInfo model_info;
   MachineConfig machine_config;
   RunConfig run_config;
+  std::basic_string<ORTCHAR_T> plugin_ep_names_and_libs;
+  std::vector<std::string> registered_plugin_eps;
+  std::string selected_devices;
+  bool list_available_devices = false;
```

Comment:
nit: call them EP devices instead. e.g.,
```suggestion
  std::string selected_ep_device_indices;
  bool list_available_ep_devices = false;
```